attributes/erlang_solutions.rb

```diff
@@ -1,5 +1,5 @@
-default['yum']['erlang_solutions']['baseurl'] = 'http://packages.erlang-solutions.com/rpm/centos/$releasever/$basearch'
-default['yum']['erlang_solutions']['description'] = 'Centos $releasever - $basearch - Erlang Solutions'
+default['yum']['erlang_solutions']['baseurl'] = "http://packages.erlang-solutions.com/rpm/centos/#{platform_version.split('.').first}/$basearch"
```

Comment:
I copied the approach from https://github.com/chef-cookbooks/yum-chef/pull/11 which was experiencing the same failure case as this Pull Request, feeling if you merged that, it was the "approved way".